### PR TITLE
Enable labeler to non-korean languages

### PR DESCRIPTION
**Pull request by electrocucaracha**
### Describe your changes

The labeler task helps to organize PR based on the folder where the changes happen. This logic needs to be applied to all languages.

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by seokho-son** [comment]
Hi @electrocucaracha 
:)

We need this PR soon, but I am not sure when to apply. 
Current condition for the branch is intentional for testing and to prevent a workflow failure by labeling.

PTAL https://github.com/cncf/glossary/issues/1322

If we apply this PR, all branches (dev-xx) have to be updated accordingly. So, all teams need to be cooperated.

(cc @yunkon-kim )

**Comment by seokho-son** [comment]
Let's bring this issue to the next Glossary working group meeting :)

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -4,8 +4,6 @@ on:
     types: [opened]
     paths:
       - 'content/**.md'
-    branches:
-      - 'dev-ko'
```
Comment (yunkon-kim) [inline .github/workflows/labeler.yml:7]:
```suggestion
    branches:
      - 'main'
      - 'dev-**'      

```

**Review by yunkon-kim — COMMENTED** [review]
Hi @electrocucaracha (CC @seokho-son)

I think it is better to specify the target branch explicitly than delete it.

`pull_request_target` has write permission so we have to be careful using it.

**Review by seokho-son — APPROVED** [review]
LGTM

Hi @jihoon-seo 
I'd like to merge this PR and disable labeler workflow manually until all dev-xx branches apply this change. (I will check update of all dev-xx branches)

Please approve :)

**Review by jihoon-seo — APPROVED** [review]
Thank you all!

**Pull request merged**